modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -224,7 +225,12 @@ public void processOpts() {
             apiTemplateFiles.put("apiImpl.mustache", "Impl.java");
             apiTemplateFiles.put("rxApiImpl.mustache", ".java");
             supportingFiles.remove(new SupportingFile("manifest.mustache", projectFolder, "AndroidManifest.xml"));
-        } else {
+        } else if ("jaxrs-spec".equals(getLibrary())) {
+        	// Don't include Jackson and GSON
+        	additionalProperties.remove("jackson");
```

Comment:
BTW, the last dependency on migbase64 is in Java/auth/HttpBasicAuth.mustache.  It has the import on com.migcomponents.migbase64.Base64.

We could possibly use the built-in Java 8 (API since 1.8) java.util.Base64, that will force users to use Java 8.

So from:
Base64.encodeToString(str.getBytes("UTF-8"), false));
To:
Base64.getEncoder().encodeToString(str.getBytes("UTF-8"))

What are your thoughts on this?  Then we could close off #2153 with 'zero' third party dependencies.